README.md

```diff
@@ -1,16 +1,169 @@
-Flutter
-=======
-
-[![Join Gitter Chat Channel -](https://badges.gitter.im/flutter/flutter.svg)](https://gitter.im/flutter/flutter?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-[![Build Status - Travis](https://travis-ci.org/flutter/flutter.svg?branch=master)](https://travis-ci.org/flutter/flutter)
-[![Build Status - AppVeyor](https://ci.appveyor.com/api/projects/status/meyi9evcny64a2mg/branch/master?svg=true)](https://ci.appveyor.com/project/flutter/flutter/branch/master)
-[![Coverage Status -](https://coveralls.io/repos/github/flutter/flutter/badge.svg?branch=master)](https://coveralls.io/github/flutter/flutter?branch=master)
+# <img src="//flutter.io/images/flutter-mark-square-100.png" alt="Flutter" width="40" height="40" /> Flutter [![Join Gitter Chat Channel -](https://badges.gitter.im/flutter/flutter.svg)](https://gitter.im/flutter/flutter?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![Build Status - Travis](https://travis-ci.org/flutter/flutter.svg?branch=master)](https://travis-ci.org/flutter/flutter) [![Build Status - AppVeyor](https://ci.appveyor.com/api/projects/status/meyi9evcny64a2mg/branch/master?svg=true)](https://ci.appveyor.com/project/flutter/flutter/branch/master) [![Coverage Status -](https://coveralls.io/repos/github/flutter/flutter/badge.svg?branch=master)](https://coveralls.io/github/flutter/flutter?branch=master)
 
 Flutter is a new mobile app SDK to help developers and designers build modern mobile apps for iOS and Android. Flutter is an alpha, open-source project.
 
-For information about using Flutter to build apps, please see the [getting started guide](http://flutter.io/getting-started/).
+### Documentation
+
+* **Main site: [flutter.io][]**
+* [Install](https://flutter.io/setup/)
+* [Get started](https://flutter.io/getting-started/)
+* [Contribute](CONTRIBUTING.md)
+
+## Fast development
+
+Flutter's <em>hot reload</em> helps you quickly
+and easily experiment, build UIs, add features, and fix
+bugs faster. Experience sub-second reload times,
+without losing state, on
+emulators, simulators, and hardware for iOS
+and Android.
+
+<img src="//flutter.io/images/intellij/hot-reload.gif" alt="Make a change in your code, and your app is changed instantly.">
+
+## Expressive, beautiful UIs
```

Comment:
It's not showing the images, plus everything's more and more shoved over to the right until "Browse the widget catalog", which is left aligned again. But yes, this is all in GitHub's preview.